content/blog/introducing-kube2pulumi/index.md

```diff
@@ -0,0 +1,417 @@
+import * as kubernetes from "@pulumi/kubernetes";
+
+const fooNamespace = new kubernetes.core.v1.Namespace("fooNamespace", {
+    apiVersion: "v1",
+    kind: "Namespace",
+    metadata: {
+        name: "foo",
+    },
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+Here, we choose Python and select our YAML manifest.
+
+```sh
+$ kube2pulumi python -f namespace.yaml
+Conversion successful! Generated File: `__main__.py`
+```
+
+```python
+import pulumi
+import pulumi_kubernetes as kubernetes
+
+foo_namespace = kubernetes.core.v1.Namespace("fooNamespace",
+    api_version="v1",
+    kind="Namespace",
+    metadata={
+        "name": "foo",
+    })
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+Here, we choose C# and select our YAML manifest.
+
+```sh
+$ kube2pulumi csharp -f namespace.yaml
+Conversion successful! Generated File: Program.cs
+```
+
+```csharp
+using Pulumi;
+using Kubernetes = Pulumi.Kubernetes;
+
+class MyStack : Stack
+{
+    public MyStack()
+    {
+        var fooNamespace = new Kubernetes.Core.V1.Namespace("fooNamespace", new Kubernetes.Types.Inputs.Core.V1.NamespaceArgs
+        {
+            ApiVersion = "v1",
+            Kind = "Namespace",
+            Metadata = new Kubernetes.Types.Inputs.Meta.V1.ObjectMetaArgs
+            {
+                Name = "foo",
+            },
+        });
+    }
+
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+Here, we choose Go and select our YAML manifest.
+
+```sh
+$ kube2pulumi go -f namespace.yaml
+Conversion successful! Generated File: main.go
+```
+
+```go
+package main
+
+import (
+        corev1 "github.com/pulumi/pulumi-kubernetes/sdk/v2/go/kubernetes/core/v1"
+        metav1 "github.com/pulumi/pulumi-kubernetes/sdk/v2/go/kubernetes/meta/v1"
+        "github.com/pulumi/pulumi/sdk/v2/go/pulumi"
+)
+
+func main() {
+        pulumi.Run(func(ctx *pulumi.Context) error {
+                _, err := corev1.NewNamespace(ctx, "fooNamespace", &corev1.NamespaceArgs{
+                        ApiVersion: pulumi.String("v1"),
+                        Kind:       pulumi.String("Namespace"),
+                        Metadata: &metav1.ObjectMetaArgs{
+                                Name: pulumi.String("foo"),
+                        },
+                })
+                if err != nil {
+                        return err
+                }
+                return nil
+        })
+}
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+## Multi-resource Example
+
+That's pretty cool but doesn't show off the full power of `kube2pulumi`. Let's try a more realistic conversion
+scenario. Our newly announced [Pulumi Kubernetes Operator] requires a few different resources to install: a Service
+Account, Role, RoleBinding, and Deployment, totaling about 130 lines of YAML. Let's try converting that directory
+to some real code!
+
+```sh
+$ ls operator
+operator.yaml  role.yaml  role_binding.yaml  service_account.yaml
+```
```

Comment:
Do these files exist somewhere we can link to so readers could try this out?  Or could we provide a URL to see the results in pulumi.com/kube2pulumi?

Reply:
https://github.com/pulumi/pulumi-kubernetes-operator/tree/master/deploy/yaml
Agree with the above suggestion to crosslink.

Reply:
Linked